public/app/core/components/SharedPreferences/SharedPreferences.tsx

```diff
@@ -43,15 +49,19 @@ const languages: Array<SelectableValue<string>> = [
   },
   {
     value: ENGLISH_US,
-    label: t('common.locale.en', 'English'),
+    label: t('common.locale.en-US', 'English'),
   },
   {
     value: SPANISH_SPAIN,
-    label: t('common.locale.es', 'Spanish'),
+    label: t('common.locale.es-ES', 'Spanish'),
   },
   {
     value: FRENCH_FRANCE,
-    label: t('common.locale.fr', 'French'),
+    label: t('common.locale.fr-FR', 'French'),
+  },
+  {
+    value: CHINESE_SIMPLIFIED,
+    label: t('common.locale.cn-Hans', 'Chinese (Simplified)'),
```

Comment:
Is the difference between `cn-Hans` and `zh-Hans` what you were talking about earlier about the difference between the locales standards in Crowdin and i18next?

Reply:
Excellent spotting. I keep forgetting whether it's zh or ch!